com.unity.render-pipelines.high-definition/Runtime/Debug/DebugDisplay.cs

```diff
@@ -169,6 +169,10 @@ public class DebugData
             public float debugOverlayRatio = 0.33f;
             /// <summary>Current full screen debug mode.</summary>
             public FullScreenDebugMode fullScreenDebugMode = FullScreenDebugMode.None;
+            /// <summary>Enable range remapping.</summary>
+            public bool enableDebugRemap = false;
```

Comment:
That's only for depth right? Shouldn't we call this with something more explicit then? enableDebugDepthRemap or something like that?